ets-demo/etsdemo/main.py

```diff
@@ -0,0 +1,27 @@
+# (C) Copyright 2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" The main function for launching the demo application.
+"""
+
+from etsdemo.app import Demo, DemoEmpty
+
+
+def _create_demo():
+    """ Create the demo object with everything setup ready to be launched. """
+    return Demo(model=DemoEmpty())
```

Comment:
This function looks silly now but this will grow and we can test this function without requiring GUI.